docs/UserGuide.md

```diff
@@ -291,7 +291,7 @@ MediContacts data are saved in the hard disk automatically after any command mod
 
 ### Editing the data file
 
-MediContacts data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcomed to update the data directly by editing that data file.
+MediContacts data are saved automatically as a JSON file `~/data/addressbook.json`, where ~ is the directory the user is running the jar file. Advanced users are welcomed to update the data directly by editing that data file.
```

Comment:
I believe we should not use `~`, as it represents home directory...
Maybe make use of `[WORKING DIRECTORY]` instead?